llama_stack/providers/remote/inference/nutanix/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Nutanix, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from .config import NutanixImplConfig
+from .nutanix import NutanixInferenceAdapter
```

Comment:
import NutanixInferenceAdapter in get_adapter_impl so llama stack build doesn't fail with a missing openai dep

Reply:
Sure!